### DISCUSSION TITLE
Viability of a searchable interface for non-photos?

### DISCUSSION THREAD

**Discussion by plk** [photoprism/photoprism#3930] [General]
I have a large corpus of drawings etc. (not photos) which have a page or so of information related to each one. I am wondering about the feasibility of attaching this information to the images and having this displayed somehow. Not sure if I could add it via EXIF or perhaps have it in the DB. I also need to attach custom search keywords to each image (EXIF?, DB?) since the AI model isn't tuned for non-photos at all. Just want to see if anyone has done anything like this before I start digging.

**Comment by graciousgrey** [comment]
PhotoPrism is reading the following metadata fields: 

https://www.photoprism.app/kb/metadata

You could use exiftool to add information to those fields. The changes will be detected by PhotoPrism during the next indexing.

See also:
https://docs.photoprism.app/user-guide/library/metadata/

**Reply by plk** [reply]
Very helpful, thanks. I need to check about size limits of the Exif description field as I need to put in several hundred words.

**Reply by graciousgrey** [reply]
You are welcome :) If you want to add search keywords I would suggest using the keywords field for this. We do not yet have a search filter for the description field.

**Reply by plk** [reply]
I don't need to filter on description, just keywords but I need the description to be visible somewhere ...

**Reply by plk** [reply]
Is there a way to get line breaks in the Description exif tag to display in the interface? Also, I would need to sorr the images in a folder by sone tag?
